context/app/static/js/pages/Dataset/hooks.ts

```diff
@@ -146,10 +147,11 @@ function getProcessedDatasetSection({
   };
 
   const sectionsToDisplay = Object.entries(shouldDisplaySection).filter(([_k, v]) => v === true);
+  const sectionTitle = pipeline ?? assay_display_name[0] ?? hubmap_id;
```

Comment:
In which cases will a dataset not have an `assay_display_name` is that for EPICs?

Reply:
I believe the `hubmap_id` was a safeguard in case of datasets without a `pipeline` value - I kept it because I wasn't sure whether all datasets were guaranteed to have an `assay_display_name`. If they are, then this can be removed.

Reply:
Ok great, it's fine to leave here I was more so curious if you ever intended for it to return the `hubmap_id`.